Application/EdFi.Ods.Common/Exceptions/InvalidConfigurationException.cs

```diff
@@ -0,0 +1,30 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using System.Runtime.Serialization;
+
+namespace EdFi.Ods.Common.Exceptions
+{
+    [Serializable]
+    public class InvalidConfigurationException : Exception
```

Comment:
I'd suggest using `ConfigurationException` from `System.Configuration` instead of introducing a custom exception for this. The [documentation](https://learn.microsoft.com/en-us/dotnet/api/system.configuration.configurationexception?view=dotnet-plat-ext-7.0&viewFallbackFrom=net-6.0) aligns with this usage.

Reply:
Good call, thanks for drawing my attention to that. Getting rusty at C#.